### ISSUE TITLE
systemd-timedated tries to parse comments as NTP services

### ISSUE THREAD

**Issue by AsciiWolf** [systemd/systemd#19043]
I am testing latest Fedora 34 with systemd-248~rc2-8.fc34.x86_64 and seeing these messages in system journal:

```
bře 18 17:56:12 fedora systemd-timedated[2404]: Failed to add NTP service "# This file is part of systemd.", ignoring: Invalid argument
bře 18 17:56:12 fedora systemd-timedated[2404]: Failed to add NTP service "# See systemd-timedated.service(8) for more information.", ignoring: Invalid argument
```

It seems that timedated is trying to parse the comments from its config file as NTP services instead of correctly ignoring them.

**Comment by yuwata** [comment]
Already fixed by #18931.

**Comment by AsciiWolf** [comment]
Oops, sorry.
